cpp/oneapi/dal/common.hpp

```diff
@@ -96,6 +96,61 @@ using v1::base;
 using v1::data_type;
 using v1::range;
 
+class result_option_id {
+    using this_t = result_option_id;
+    using bitset_t = std::uint64_t;
+
+public:
+    result_option_id() = default;
+    result_option_id(const bitset_t& mask) : mask_{ mask } {}
+
+    operator bool() const {
+        return mask_ > 0ul;
+    }
+
+    const bitset_t& get_mask() const {
+        return mask_;
+    }
+
+    static result_option_id get_result_id_by_index(std::int64_t result_index) {
+        return this_t{}.set_mask(std::uint64_t(1) << result_index);
+    }
+
+private:
+    result_option_id& set_mask(const bitset_t& mask) {
+        this->mask_ = mask;
+        return *this;
+    }
+
+    bitset_t mask_;
+};
+
+template <typename ResultOptionIdType>
+constexpr inline bool is_result_option_id_v =
+    std::is_base_of_v<result_option_id, ResultOptionIdType>;
```

Comment:
Add extra space, it's not readable

Reply:
Its done according to clang-format. won't fix

Reply:
No, it's not. Clang-format removes blank lines only if it's more than one. It sounds strange as for other declarations, you have at least one line.

Reply:
fixed